test/test_util.py

```diff
@@ -227,24 +229,22 @@ def test_least_nonzero_2():
     )
 
 
-def test_gdal_extent():
+def test_rio_extent():
     # Create a simple georeferenced test file
-    ds = gdal.GetDriverByName('GTiff').Create('test.tif', 11, 11, 1, gdal.GDT_Float64)
-    ds.SetGeoTransform((17.0, 0.1, 0, 18.0, 0, -0.1))
-    band = ds.GetRasterBand(1)
-    band.WriteArray(np.random.randn(11, 11))
-    srs = osr.SpatialReference()
-    srs.ImportFromEPSG(4326)
-    ds.SetProjection(srs.ExportToWkt())
-    ds = None
-    band = None
+    with rasterio.open("test.tif", mode="w",
+                       width=11, height=11, count=1,
+                       dtype=np.float64, crs="EPSG:4326",
```

Comment:
I think you might get warnings about using a crs as a string. I use

```
from rasterio.crs import CRS

crs = CRS.from_epsg(4326)
```

Reply:
Done.